docs/system-admin-guide/time-and-costs/README.md

```diff
@@ -11,7 +11,24 @@ Navigate to *Administration* -> *Time and costs*. Here you can configure costs i
 
 ## Default time and cost settings
 
-Navigate to *Administration* -> *Time and costs* -> *Defaults* to define settings for costs in OpenProject.
+To define default settings for logging time and costs navigate to *Administration* -> *Time and costs* -> *Defaults* and select *Time* or *Costs* tab respectively.
+
+### Define default time settings (Enterprise add-on)
+
+> [!NOTE] 
+> Enforcing exact time entries for time logging is an Enterprise add-on and can only be used  with [Enterprise cloud](https://www.openproject.org/docs/enterprise-guide/enterprise-cloud-guide) or [Enterprise on-premises](https://www.openproject.org/docs/enterprise-guide/enterprise-on-premises-guide). An upgrade from the free Community edition is easily possible.
+
+Under the *Time* tab you can define which fields are required to be fill out when logging time across all projects. 
+
+- **Allow exact times tracking**: enables start and finish time fields for time logging. If disabled, only the amount of time spent can be logged, but not the start or end times. 
```

Comment:
We need to update this labels after the bug fix: https://community.openproject.org/projects/openproject/work_packages/63788